target/linux/ramips/mt7620/config-5.15

```diff
@@ -153,7 +155,6 @@ CONFIG_PHYLIB=y
 # CONFIG_PHY_MT7621_PCI is not set
 CONFIG_PHY_RALINK_USB=y
 CONFIG_PINCTRL=y
-# CONFIG_PINCTRL_AW9523 is not set
```

Comment:
This is a downstream driver for one mt7621 device, so we still need it `is not set` for all the other subtargets (CI is failing for this).


https://github.com/openwrt/openwrt/blob/3556455f040341099a6563968a6e9e8c22c0847e/target/linux/ramips/dts/mt7621_tenbay_t-mb5eu-v01.dts#L74

https://github.com/openwrt/openwrt/blob/3556455f040341099a6563968a6e9e8c22c0847e/target/linux/ramips/files/drivers/pinctrl/pinctrl-aw9523.c#L1107

https://github.com/openwrt/openwrt/blob/3556455f040341099a6563968a6e9e8c22c0847e/target/linux/ramips/patches-5.15/805-pinctrl-AW9523.patch#L42

Reply:
Thank you, I noticed it. It was dropped by `make kernel_oldconfig`. I'll re-add it.